feat: 3D geometrier for KL åk type 14-2

Wije0410 opened the pull request:
Lagt til 16 3D geometrier for KL åk type 14-2 forsterket, fra 28-34m i lengde.

Wije0410 committed:
feat: 3D geometrier for KL åk type 14-2

Lagt til 16 3D geometrier for KL åk type 14-2 forsterket, fra 28-34m i lengde.

Wije0410 committed:
fix: KL åk type 14-2 28m har blitt rejustert

Den venstre enderammen i kL åk type 14-2 28m var ikke lagt riktig med den venstre siden av mellomrammen. Venstre endreramme er nå justert til å ligge riktig.

clfey reviewed (changes requested):
1. Slett alle filer som ikke er DWG / som er uvedkommende i 3D\2025.1\EH
<img width="451" height="156" alt="image" src="https://github.com/user-attachments/assets/b423cdb6-237b-482b-a3fc-38a84eae0b77" />

2. Legg de faktiske kildefilene til i Onedrive \ RC_3D\NO-BN\EH under en DWG-katalog for 14-2 når du er ferdig. Dette er den EGENTLIGE lagringsplassen vår for 3D. Github-repo'et er det vi velger å sende ut til kundene.

3. Galt filnavn på alle unntatt 29-meter lengde. Det er et løpenummer i filnavnet - sjekk hva som er gjort for 12-1 og 12-2 og 14-1.

4. Åk 14-2 på 28M lengde har en grafikk-feil:
<img width="390" height="416" alt="image" src="https://github.com/user-attachments/assets/4577bed9-db97-4af6-ab23-adbe00cc1fb4" />

5. 14-2, 28M:
Det skal alltid være vertikalstag i skjøtepunktene mellom rammene.
<img width="866" height="234" alt="image" src="https://github.com/user-attachments/assets/6bf7b741-582f-44f6-bbb6-6d231aff1215" />

6. Det skal være 4 mm toleranse for bygging mellom nabo-rammer. Da skal man kunne se tvers gjennom alle seks hull i den nedre skjøtelasken (90mm).
Studér plassering av rammene i et par 12-2 tegninger så du skjønner hvordan du får 4 mm ekstra forskyvning for hver ny ramme du legger til.
<img width="866" height="412" alt="image" src="https://github.com/user-attachments/assets/66d38646-259c-4117-a31f-22292f4f3845" />

7. Før du CLOSE'er filen så skal du allid:
* Kjøre PURGE /All / Nested
* Klikke i NAVVCUBE-senteret for å re-posisjonere og skalere i hver av de fire viewportene.
<img width="1914" height="1411" alt="image" src="https://github.com/user-attachments/assets/69a9b33e-9867-40f9-95c6-1a3bec3c3204" />

8. De fire ATTDEF'ene skal plassert eksakt på de posisjonene som rfemgår av 12-2 rammer og 12-2 sammenstilte tegninger.
<img width="706" height="455" alt="image" src="https://github.com/user-attachments/assets/ff984862-2f41-4ae3-9c30-3df606a5aa50" />

Ikke slik:
<img width="316" height="256" alt="image" src="https://github.com/user-attachments/assets/efd61076-b765-48c5-ada9-d7debb1cf0a8" />

9. Alle rammer skal ha blokknavn - dvs filnavn - iht samme navngivingsfilosofi som er benyttet for 12-2 rammer. Rammene skal ligge som egne filer med slike filnavn på RC_3D/NO-BN/EH/EH-AAK sammen med selve de sammensatte åkene.
<img width="810" height="927" alt="image" src="https://github.com/user-attachments/assets/ea893e40-2222-4436-823d-9fa707ebe5be" />

Wije0410 committed:
fix: reviderte 3D geometrier for KL-åk 14-2

Revisjon gjort på forsterket KL-åk type 14.

clfey commented:
- [ ] Lag sjekklisten for alt som skal kontrolleres, ta med nye sjekkpunkter om nødvendig basert på kommentarene nedenfor. Vis sjekklisten til Claus.

* Jeg mener at det bare var LEFT view som skal vise en tekst som sier hva slags ramme det er snakk om. Det er dels for å ha standardisert dette, dels fordi jeg mener å huske at 14-1 er gjort på det viset, og dels for å unngå at man glemmer å oppdatere den andre teksten ene dersom en en av dem korrigeres.

* MEN nå står de der, så du velger om du vil korrigere begge der det trengs eller om du vil fjerne den på HØYRE view. CLFEY mener at det skal være bare EN slik tekst som angir rammetype.

* CLFEY oppdaget tekster som står for langt unna overflaten eller for nær, og i ett tilfelle på INNSIDEN av åket. Tekstplassering skal altså være X=-0.351 eller X=+0.351. Utover dette skal Y og Z-korrdinat være identisk for samtlige tektser, og type justering skal være identisk.

To enderammer og seks mellomrammer har blitt kontrollert.

Disse rettes og leveres på nytt for PR godkjenning, sammen med oppdaterte sammensatte åk.

HSIDE 7M mellomramme har fått "DNA" i seg, innholdet skal flyttes over til en fil som ikke har DNA. 
CLFEY har utført dette allerede på RC_3D ... EH-AAK . DWG2018 - se nytt lagringstidspunkt for denne filen.

<img width="1314" height="699" alt="image" src="https://github.com/user-attachments/assets/f6a8e844-8033-4a3a-93d7-67b0f0a529b1" />

Mellomramme VSIDE 6M har feil tekst ("høyre").
CLFEY har kjorrigert på RC_3D.
<img width="1575" height="916" alt="image" src="https://github.com/user-attachments/assets/ee11b805-7782-41a5-a667-178d0c9008b0" />

Mellomramme VSIDE 6M har "baksideteksten" plassert inni åket
CLFEY har kjorrigert på RC_3D.
<img width="1436" height="1002" alt="image" src="https://github.com/user-attachments/assets/60e26a6e-f295-46f2-8381-59d98a26b9c3" />

Mellomramme VSIDE 7M (og gjelder trolig alle den- og mellomrammer)
TEKSTER i form av TEXT og MTEXT skal "sveve" 1 mm utenfor overfalten de tilhører. 
Dersom teksten ligger for langt unna så blir det "ustandard" og kan i verste fall bli rart å se på. Dersom den ligger helt inntil så vil ikke Navisworks og andre visningsprogrammer klare å skille tekst fra bakenforliggende overflate. Man kan ikke anta at DrawOrder blir overført til ekstern programvare.
Claus har IKKE rettet dette på RC_3D, alle filene skal gjennomgås på nytt.
<img width="672" height="472" alt="image" src="https://github.com/user-attachments/assets/fd9581df-e5be-4a58-a237-25d9eb1d68c7" />

Wije0410 committed:
fix: plassering av tekst som beskriver ende- og mellomramme navn

Teksten som viser til navn på ende- og mellomramme er plassert på likt sted for hvert 3D-objekt.

clfey commented:
TIL SJEKKLISTE - og rette opp:

Text posision - være systematisk og brulk pene verdier / logiske verdier:
<img width="291" height="102" alt="image" src="https://github.com/user-attachments/assets/5a11c515-3034-41c6-87b0-07b46bb4fa69" />
<img width="272" height="97" alt="image" src="https://github.com/user-attachments/assets/632543d3-7c83-4d55-b345-dc9dc52d2475" />


ATTDEF plasseringer - ente 5 cm under eller 5 cm over innsettingspunktet, avhengig av hvor grafikkens SOLIDS osv er hen.
0,.000 0.020, -0.050 COPYRIGHT
0,.000 0.040, -0.050  ARTIKKEL
0,.000 0.060, -0.050 ARTIKKELNUMMER
0,.000 0.080, -0.050  KILDE 


UNITS settings skal være:
<img width="374" height="448" alt="image" src="https://github.com/user-attachments/assets/92f95632-2ff7-43d8-9726-8bf2bc14460b" />


Bruk PURGE all, deretter Ctrl+1 (PROPERTIES) og Ctrl+A (_AI_SELECTALL) og trykk på nedpil for antall entities for å se om du har noen uønskede ting som ligger igjen og skulle vært slettet.
<img width="659" height="340" alt="image" src="https://github.com/user-attachments/assets/dee16c18-1650-4088-9f6c-b53076f15c08" />

Wije0410 committed:
fix: renset sammensatt åk og rejustert plassering på tekst

Delene som bygger opp åkene har blitt renset for unødvendige objekter.  Plassering på tekst har også blitt endret til runde verdier. Avrunding for lengder og vinkler har justert til 3 desimaler.
